### DISCUSSION TITLE
Stepper Vertical

### DISCUSSION THREAD

**Discussion by BiancaMach** [Telefonica/mistica-design#1840] [Proposals]
### Goal

Discutir a possibilidade de evolução do componente Stepper, adicionando uma variação vertical.

### Context

Há alguns meses, uma das squads do time Vivo demandou um stepper vertical para uma jornada e trouxemos a vocês, mas como era algo específico não foi adiante para ser incluído ao Mística e agora tivemos mais um time demandando esse componente. Foi comentado que a O2 também solicitou o mesmo componente há um tempo atrás. Assim, gostaríamos de avaliar a possibilidade de adicionar essa variação de Stepper Vertical ao componente de Stepper.

### Impact

A evolução desse componente manteria a consistência das jornadas que já utilizavam uma solução parecida no design anterior do aplicativo Vivo. Além de padronizar esse tipo de fluxo na jornadas e melhorar a experiência do usuário.

### Discussion Questions

N/D

### Do you have a visual proposal?

https://www.figma.com/design/XwL4DYDJYLokQxHuhI4DyB/POC---Stepper-Vertical?node-id=331-5823&t=6mp2SUftFbhDKqsd-1

**Comment by aweell** [comment]
Hola @BiancaMach, contesto a esta discussion en el [hilo original](https://github.com/Telefonica/mistica-design/discussions/1055), para otras ocasiones si ya existe un hilo al respecto, siéntete libre de reabrirlo sin problema, así centralizamos toda la información en un mismo punto. Muchas gracias 
 
Cierro aquí y la discussion continua ahí.
